pyspecdata/core.py

```diff
@@ -7552,6 +7557,7 @@ def _init_datanode(self, datanode, **kwargs):
                     )
                 datadict["axes"].pop(axisname)
             kwargs.update({"axis_coords": myaxiscoords})
+            kwargs.update({"axis_coords_units": myaxis_units})
```

Comment:
To make sure that this is being done correctly, I am pretty sure that there is an example that writes and reads an hdf file.. this should involve modifying that example as well

Reply:
tested and passed

Reply:
Btw sorry I didn't see this comment:

Which file are we talking about here?  I'm specifically asking that you should modify the existing example so that it checks that it can see the units that were saved to file.

Reply:
@guinneak198 can you please update the example, so that we can merge this PR??